semantic/constraints.go

```diff
@@ -358,7 +364,7 @@ func (v ConstraintGenerator) typeof(n Node) (PolyType, error) {
 		}
 		v.cs.AddKindConst(tv, ObjectKind{
 			properties: map[string]PolyType{n.Property: ptv},
-			lower:      LabelSet{n.Property},
+			lower:      LabelSet{},
```

Comment:
I was going to keep it because it would be useful if we ever wanted to make object literals require certain properties. If we end up deciding against it, we can remove this.